manifests/repo.pp

```diff
@@ -1,4 +1,56 @@
+# hashi_stack::repo
```

Comment:
does that provide any benefit / does an editor or puppet-strings pick that line up?

Reply:
not really started out as a placeholder which i should have removed